docs/querying.md

```diff
@@ -16,6 +16,15 @@ Post.where(:created_at, :gt, Time.local - 7.days)
 
 Supported operators are :eq, :gteq, :lteq, :neq, :gt, :lt, :nlt, :ngt, :ltgt, :in, :nin, :like, :nlike
 
+Alternatively, where accepts a raw SQL clause, with "?" as a placeholder to avoid SQL Injection.
+```crystal
+Post.where(:created_at, :gt, Time.local - 7.days)
+  .where("LOWER(author_name) = ?", name)
+  .where("tags @> '{"Journal", "Book"}') # PG's array contains operator
```

Comment:
Related to the above comment.  Since this is using PG specific behavior, the other placeholder should probably be updated to match.

Reply:
Replied above.